backend/.env

```diff
@@ -1 +1,2 @@
-DATABASE_URL = "file:./dev.db"
\ No newline at end of file
+DATABASE_URL = "file:./dev.db"
+SESSION_SECRET = '6qyI+87A_?=;dw'
```

Comment:
this is still showing up in the diff. you might need to run something like https://www.wisdomgeek.com/development/how-to-permanently-remove-a-file-from-git-history/

Reply:
followed all the steps. not sure how to check if it worked though